tests/expr_and_series/reduction_test.py

```diff
@@ -26,7 +26,6 @@
             {"a": [1, 2, 3], "min": [4, 4, 4]},
         ),
     ],
-    ids=range(5),
```

Comment:
why remove these?

Reply:
It was unused in this test

Reply:
they are pytest ids https://docs.pytest.org/en/stable/example/parametrize.html

though admittedly `range(5)` aren't very helpful ids 😄

Reply:
I know, most likely I introduced them😂  I spotted they were unused while I was introducing for a new test case (I did not recall the syntax, but I knew we had it in our test suite).

They where previously used to xfail certain tests via the id value identifier. I believe that once you added full support for aggregations in pyspark and duckdb the xfail-ures where removed and these ids unused!